x-pack/legacy/plugins/infra/public/pages/logs/index.tsx

```diff
@@ -152,3 +113,25 @@ export const LogsPage = injectUICapabilities(({ match, uiCapabilities }: LogsPag
     </Source.Context.Provider>
   );
 });
+
+const pageTitle = i18n.translate('xpack.infra.header.logsTitle', {
```

Comment:
This deduplicates some translations and removes the repeated translation from the render cycle.